omero/sysadmins/config.txt

```diff
@@ -1180,13 +1180,23 @@ Default: `pixelDataEventLogQueue`
 omero.pixeldata.max_plane_height
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+With :property:`omero.pixeldata.max_plane_width` specifies
+the plane size cutoff above which a pixel pyramid should
+either be read directly from the file format or generated
+by the pixeldata service.
```

Comment:
That's a good question. I could rephrase this and limit the scope to the server aspect of things e.g. `the plane size cutoff above which a pixel pyramid will be generated by the pixeldata service unless subresolutions can be read from the file format`.

When it comes to reading, I think  services like rendering are re-using the same cutoff to determine whether an image has pyramidal levels or not. Hence for our own clients, your statement should be true. That being said, it might be possible to use the low-level API to access subresolutions if they exist.